faust/utils/terminal/tables.py

```diff
@@ -58,3 +70,22 @@ def logtable(data: TableDataT,
 
 def _get_best_table_type(tty: bool) -> Type[Table]:
     return SingleTable if tty else AsciiTable
+
+
+def dict_as_ansitable(d: Mapping,
+                      *,
+                      key: str = 'Key',
+                      value: str = 'Value',
+                      sort: bool = False,
+                      sortkey: Callable[[Any], Any] = itemgetter(0),
+                      target: IO = sys.stdout,
```

Comment:
@ask It doesn't seem that we are using `target` in the body of the function, should it be removed from the parameters?

Reply:
`        target (IO): Target should be the destination output file
                     for your table, and defaults to :data:`sys.stdout`.
                     ANSI codes will be used if the target has a controlling
                     terminal, but not otherwise, which is why it's important
                     to pass the correct output file.`

Target is used only to check if output is a `tty`. We don't want to use ANSI codes if this is something that will be logged.

Reply:
Ah, yeah. You're correct! I fixed it now